documentation/source/user_section/tutorials/registration-to-template/rootlets-based-registration.rst

```diff
@@ -0,0 +1,14 @@
+.. _rootlets-based-registration:
+
+Registering images to the PAM50 template based on spinal nerve rootlets
+#######################################################################
+
+This tutorial demonstrates how to `register <http://jpeelle.net/mri/image_processing/registration.html>`_ an anatomical MRI scan to the PAM50 template based on the spinal nerve rootlets instead of the vertebral levels. While T2w images are used for this tutorial, the registration is applicable also to other contrasts; see `sct_deepseg -rootlets` for the list of supported contrasts for automatic rootlet segmentation.
```

Comment:
Markdown uses single-backtick, while .rst uses double (I mess this up all the time!).

```suggestion
This tutorial demonstrates how to `register <http://jpeelle.net/mri/image_processing/registration.html>`_ an anatomical MRI scan to the PAM50 template based on the spinal nerve rootlets instead of the vertebral levels. While T2w images are used for this tutorial, the registration is applicable also to other contrasts; see ``sct_deepseg -rootlets`` for the list of supported contrasts for automatic rootlet segmentation.
```